sections/03_categorize.md

```diff
@@ -44,92 +44,76 @@ regards to data availability and labeling.
+melanoma, the deadliest form of skin cancer, from
+dermoscopic [@tag:Codella2016_ensemble_melanoma
+@tag:Yu2016_melanoma_resnet] and non-dermoscopic images of
+skin lesions [@tag:Jafari2016_skin_lesions @tag:Esfahani2016_melanoma
+@tag:Esteva2017_skin_cancer_nature]as well as age-related macular degeneration
+[@tag:Burlina2016_amd]. Pre-training on natural
+images can enable very
+deep networks to succeed without overfitting. For the melanoma task, reported performance was competitive with or better
+than a board of certified dermatologists
+[@tag:Codella2016_ensemble_melanoma @tag:Esteva2017_skin_cancer_nature].
+
+The reuse of features from natural images is also growing for
+radiographic images, where datasets are often too small to train large deep neural networks without
+these techniques [@tag:Bar2015_nonmed_tl @tag:Shin2016_cad_tl
+@tag:Rajkomar2017_radiographs @tag:Lakhani2017_radiography]. Rajkomar et al.
 [@tag:Rajkomar2017_radiographs] showed that a deep CNN trained on natural images
-can boost performance in radiology image classification. However, the target
+boosts performance in radiographic images. However, the target
 task required either re-training the initial model from scratch with special
 pre-processing or fine-tuning of the whole network on radiographs with heavy
-data augmentation to avoid overfitting. Shin et al. [@tag:Shin2016_cad_tl]
+data augmentation to avoid overfitting.
+
+The technique of reusing features from a different task falls into the broader area of
+transfer learning, which we devote a section to in the discussion.
+Though we've mentioned numerous successes for the transfer of natural image features to new tasks,
+we expect that a lower proportion of negative results would be published.
+The analysis of magnetic resonance
+images (MRIs) is also faced with the challenge of small training sets. In this
+domain, Amit et al. [@tag:Amit2017_breast_mri] investigated
+the tradeoff between pre-trained models from a different domain and
+a small CNN trained only with MRI images. In contrast with the other selected literature, they found a smaller network
+trained with data augmentation on few hundred images from a few dozen
+patients can outperform a pre-trained out-of-domain classifier. Data augmentation is a different strategy to deal with small training
+sets. The practice is exemplified by a series of papers that analyze images
+from mammographies [@tag:Dhungel2015_struct_pred_mamm @tag:Dhungel2016_mamm
+@tag:Zhu2016_mult_inst_mamm @tag:Zhu2016_advers_mamm
+@tag:Dhungel2017_mamm_min_interv]. To expand the number and diversity of images,
+researchers constructed adversarial examples
+[@tag:Zhu2016_advers_mamm]. Adversarial examples are
+constructed by applying a transformation that changes training images but not
+their content - for example by rotating an image by a random amount. An alternative in the domain is to train towards human-created features before
+subsequent fine tuning [@tag:Dhungel2016_mamm].
```

Comment:
Do we think training toward human-centric features is a good idea?  In Study, we discourage that practice or suggest it is likely to diminish in the future.